README.md

```diff
@@ -8,7 +8,7 @@ Library checks for updates using configuration from remote or local resource.
 ## Getting via jcenter
 
 ```groovy
-implementation 'co.infinum:prince-of-versions:latest_version'
+implementation 'co.infinum:prince-of-versions:3.0.0'
```

Comment:
Did you manually update this value or is there a gradle task for this? GoldenEye and Goldfinger have automatic tasks that update this value so you can check it there

```groovy
task generateReadme {
    doFirst {
        def readmeFile = new File("${project.rootDir}/README.md")
        def content = readmeFile.text
        content = content.replaceAll("'co\\.infinum:goldeneye:.+?'", "'co.infinum:goldeneye:${versions.goldeneye}'")
        readmeFile.setText(content)
    }
}
```

Reply:
Awesome